docs/user/03-11-customize_secret.md

```diff
@@ -0,0 +1,38 @@
+# Customize Credentials and Access
+
+To use your own credentials as the default and manage your cluster on your own, customize the `sap-btp-manager` Secret.
+
+## Context
+
+When you create SAP BTP, Kyma runtime, the `sap-btp-manager` Secret is automatically created as the default Secret managing the SAP BTP service operator's resources. 
+Any changes to the Secret are reverted, and the previous settings are restored within up to 24 hours.
+See [Preconfigured Credentials and Access](03-10-preconfigured-secret.md).
+
+To stop the reconciliation of the `sap-btp-manager` Secret, you must customize it. With the customized Secret, you can perform the following actions:
```

Comment:
I see this the other way around.
To customize you must stop the reconciliation.